src/ExpressReceiver.ts

```diff
@@ -40,7 +39,7 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
     this.server = createServer(this.app);
 
     const expressMiddleware: RequestHandler[] = [
-      verifySlackRequest(signingSecret),
+      ExpressSignatureVerifier.create(signingSecret),
```

Comment:
If you prefer other names, please let me know. I don't have any strong opinions on this.